docs/openrpc.json

```diff
@@ -1015,120 +1031,6 @@
-                "type": ["string", "array"],
-                "items": {
-                  "type": "string"
-                }
-              },
-              "topics": {
-                "type": ["array"],
-                "items": {
-                  "type": ["string", "null"]
-                }
-              }
-            }
-          },
-          "description": "Filter options to specify the exact event(s) logs to get."
-        }
-      ],
-      "result": {
-        "name": "filter_id",
-        "schema": {
-          "type": "string"
-        },
-        "description": "The ID of the newly created filter."
-      }
-    },
-    {
-      "name": "eth_uninstallFilter",
-      "summary": "Uninstalls a filter with given ID. Should always be called when watch is no longer needed.",
-      "description": "![](https://raw.githubusercontent.com/hashgraph/hedera-json-rpc-relay/main/docs/images/http_label.png)",
-      "params": [
-        {
-          "name": "filter_id",
-          "required": true,
-          "schema": {
-            "type": "string"
-          },
-          "description": "The ID of the filter to uninstall."
-        }
-      ],
-      "result": {
-        "name": "uninstalled",
-        "schema": {
-          "type": "boolean"
-        },
-        "description": "True if the filter was successfully uninstalled, otherwise false."
-      }
-    },
-    {
-      "name": "eth_getFilterChanges",
-      "summary": "Polling method for a filter, which returns an array of logs which occurred since last poll.",
-      "description": "![](https://raw.githubusercontent.com/hashgraph/hedera-json-rpc-relay/main/docs/images/http_label.png)",
-      "params": [
-        {
-          "name": "filter_id",
-          "required": true,
-          "schema": {
-            "type": "string"
-          },
-          "description": "The ID of the filter to check for changes."
-        }
-      ],
-      "result": {
-        "name": "logs",
-        "schema": {
-          "type": "array",
-          "items": {
-            "$ref": "#/components/schemas/Log"
-          }
-        },
-        "description": "Array of log objects since last poll."
-      }
-    },
-    {
-      "name": "eth_getFilterLogs",
-      "summary": "Returns an array of all logs matching filter with given ID.",
-      "description": "![](https://raw.githubusercontent.com/hashgraph/hedera-json-rpc-relay/main/docs/images/http_label.png)",
-      "params": [
-        {
-          "name": "filter_id",
-          "required": true,
-          "schema": {
-            "type": "string"
-          },
-          "description": "The ID of the filter to retrieve all matching logs."
-        }
-      ],
-      "result": {
-        "name": "logs",
-        "schema": {
-          "type": "array",
-          "items": {
-            "$ref": "#/components/schemas/Log"
-          }
-        },
-        "description": "Array of log objects that match the filter."
-      }
-    },
```

Comment:
qq why are we removing these?

Reply:
the schema for those methods was written twice; here we're removing one of them --> compared both to ethereum specification and left the one consistent with it

Reply:
nice thanks for explaning!